graphql/resolvers.go

```diff
@@ -3048,7 +3060,7 @@ func (r *taskResolver) CanSchedule(ctx context.Context, obj *restModel.APITask)
 	if err != nil {
 		return false, err
 	}
-	return *canRestart == false && !obj.Aborted, nil
+	return !utility.FromBoolPtr(canRestart) && !obj.Aborted, nil
```

Comment:
This seems like an unrelated change but it seems better to just have canRestartTask not return a pointer

Reply:
Yeah this is unrelated i made this change  to suppress a random warning i saw while scrolling through this file. Theres a ticket tracking refactoring that function here. https://jira.mongodb.org/browse/EVG-15698